.github/workflows/gluwa.yml

```diff
@@ -153,46 +137,85 @@ jobs:
       - uses: Swatinem/rust-cache@v2
 
       - name: Run tests
-        uses: actions-rs/cargo@v1
+        uses: gluwa/cargo@dev
         with:
           command: test
-          args: --features runtime-benchmarks
+          args: --release --features runtime-benchmarks
```

Comment:
this change is breaking

Reply:
Also confirmed in https://github.com/gluwa/polkadot-sdk/actions/runs/11820013472/job/32931491897?pr=4 when I cherry-pick the same commit into PR#4.